test/process.test.js

```diff
@@ -103,4 +105,20 @@ describe(`process`, function () {
       //# sourceMappingURL=data:application/json;base64,eyJ2ZXJzaW9uIjozLCJzb3VyY2VzIjpbIjxhbm9uPiJdLCJzb3VyY2VzQ29udGVudCI6WyI8dGVtcGxhdGU-SGk8L3RlbXBsYXRlPiJdLCJuYW1lcyI6W10sIm1hcHBpbmdzIjoiO0FBQUEsZUFBQSxTQUFVLENBQUEsRUFBRSxDQUFBLEVBQUE7SUFBQTtRQUFBLE9BQUEsS0FBQSxTQUFBLENBQUEsRUFBVztJQUFEO0FBQUEsR0FBQyJ9`
     );
   });
+
+  it("handles 'declarative shadow-dom'", function () {
+    // https://developer.chrome.com/docs/css-ui/declarative-shadow-dom
+    let input = `
+<template>
+  <template shadowrootmode="open">
```

Comment:
`<template>` is _only_ useful with the shadowrootmode attribute, otherwise `<template>` isn't rendered at all

![image](https://github.com/embroider-build/content-tag/assets/199018/c971fd92-51c6-419d-a1b6-8c523e1b1ebf)